bundles/org.openhab.binding.boschshc/src/main/java/org/openhab/binding/boschshc/internal/devices/bridge/LongPolling.java

```diff
@@ -166,8 +166,9 @@ public void onComplete(@Nullable Result result) {
                 // NOTE: This handler runs inside the HTTP thread, so we schedule the response
                 // handling in a new thread because the HTTP thread is terminated after the
                 // timeout expires.
-                scheduler.execute(() -> longPolling.onLongPollComplete(httpClient, subscriptionId, result,
-                        this.getContentAsString()));
+                scheduler.execute(() -> {
```

Comment:
Please revert to the original version without the curly braces since we now have the Sonar issue `remove useless curly braces around statement` (rule `java:S1602`)

Reply:
Don't see that issue/waning in vscode, or where should i see this?
Removed the braces anyway, thanks.

Reply:
Try installing the `SonarLint` extension, it is a brilliant tool that helped me to find and fix many issues, and as a "side effect" to even learn a lot about Java and other languages :+1: